### PR TITLE
Replace event starting soon alert based on wireframe revisions.

### PR DESCRIPTION
**Pull request by maurizi**
Connects to #1358

### CODE REVIEW

**Comment by jwalgran** [comment]
The banner on the event page says "starting soon" rather than "this event has started" when the event is in progress

This screen shot was taken May 1 at noon.

screen shot 2015-05-01 at 8 59 19 am

**Comment by jwalgran** [comment]
If I click RSVP on an event in progress, I don't see the check in button until I refresh the page. 

screen shot 2015-05-01 at 9 02 21 am

**Comment by jwalgran** [comment]
Edge case. If you remove someone's check in after they have loaded the dashboard/event page, the start mapping button takes them to a generic 403 page. It is not worth a lot of effort to fix this, but if it is possible to easily redirect to the event check in page rather than a forbidden URL it would be a bit more person friendly.

File: src/nyc_trees/apps/event/templates/event/event_detail.html [hunk 1]
```diff
@@ -9,23 +9,22 @@
...
-                        class="btn btn-switch pull-right">Start Mapping</a>
-                    {% endif %}
+    <div class="event-alert">
+        <div class="container">
+            <div class="row">
+                <div class="event-alert-text col-md-8 col-md-offset-1 col-xs-8">
+                    This event is
                     {% if event.in_progress %}
-                      <p>This event is currently in-progress!</p>
+                        starting soon.
```
Comment (jwalgran) [inline src/nyc_trees/apps/event/templates/event/event_detail.html]: This conditional is backwards

**Comment by jwalgran** [comment]
The only bit of styling that does not match the wireframes is the lack of vertical centering on the action button.

screen shot 2015-05-01 at 9 33 38 am

File: src/nyc_trees/apps/home/templates/home/partials/dashboard.html [hunk 2]
```diff
@@ -3,6 +3,25 @@
...
 {% url 'events_list_page' as events_page_url %}
 {% url 'progress_page' as progress_page_url %}
+{% if event %}
+    <div class="event-alert">
+        <div class="container">
+            <div class="row">
+                <div class="event-alert-text col-xs-8">
+                    The event {{ event.title }} for {{ event.group.name }} is
+                    {% if event.in_progress %}
+                        starting soon.
```
Comment (jwalgran) [inline src/nyc_trees/apps/home/templates/home/partials/dashboard.html]: This conditional is backwards

File: src/nyc_trees/nyc_trees/context_processors.py [hunk 3]
```diff
@@ -32,16 +16,6 @@ def user_settings_privacy_url(request):
 
 
 def config(request):
-    # At the time this function was written, the generated context was
-    # only used in the base.html template, and our AJAX requests all
-    # render partials, not full templates inheriting from base. Given
-    # those constraints, we can skip generating this context, which
-    # saves some database query time.
-    if request.is_ajax():
```
Comment (jwalgran) [inline src/nyc_trees/nyc_trees/context_processors.py]: Removing this broke a unit test

**Comment by maurizi** [comment]
@jwalgran I think all of the issues you raised have been addressed, can you take another look?

**Comment by jwalgran** [comment]
The event admin does not see a "Check-in Guests" button unless they RSVP. Not sure if this is by design.

screen shot 2015-05-01 at 10 46 35 am

screen shot 2015-05-01 at 10 47 13 am

**Comment by jwalgran** [comment]
+1 I tried various combinations of user, event time, and RSVP status and saw all the appropriate buttons.

**Comment by maurizi** [comment]
@jwalgran The issue with  "Check-in Guests" is not by design, but is kind of tricky to deal with properly.  I'm inclined to leave it as-is for now.

**Pull request merged**